apps/pgsql/patroni/openshift/deployment.yaml

```diff
@@ -309,7 +319,7 @@ objects:
   kind: RoleBinding
   metadata:
     labels:
-      cluster-name: ${INSTANCE}
```

Comment:
We can also remove the 'INSTANCE' from the list of parameter

Reply:
Oddly my copy has this removed... not sure whats happening there, will double check after the next commit.

Reply:
nm, found it and removed it.